test/unit/money_test.rb

```diff
@@ -43,7 +43,7 @@ class MoneyTest < ActiveSupport::TestCase
 
     test "should not accept numbers too big" do
       assert_raises(SmartAnswer::InvalidResponse) do
-        SmartAnswer::Money.new 1_111_111_111_111_111_122_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_225_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_222_222_222_222_222_222_222_211_111_111_111_111_111_133_333_333_333_333_333_333_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_444_444_444_444_444_444_444_444_444_448_888_888_888_888_888_888_888_888_888_888_888_888_888_888_888_888_888_888_888_887_777_777_777_777_778_888_888_888_888_889_999_999_999.0
+        SmartAnswer::Money.new 1_111_111_111_111_111_122_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_222_225_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_222_222_222_222_222_222_222_211_111_111_111_111_111_133_333_333_333_333_333_333_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_555_444_444_444_444_444_444_444_444_444_448_888_888_888_888_888_888_888_888_888_888_888_888_888_888_888_888_888_888_888_887_777_777_777_777_778_888_888_888_888_889_999_999_999.0 # rubocop:disable Lint/FloatOutOfRange
```

Comment:
I'm not a fan of disabling linting rules in line instead of addressing them. 
Alternatively we could use BigDecimal instead of Float here or update the value in the test (but we don't have enough product knowledge).